mac-build-support-lib.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+# This script helps you to build apple fat binaries.
+# This uses the excellent https://github.com/leetal/ios-cmake cmake platform file.
```

Comment:
Can you document somewhere exactly which version of that file was incorporated (by version if that exists, or else a git commit).

Reply:
I'll add the sha-1 of the file inside the script.